defaults/.golangci.yaml

```diff
@@ -0,0 +1,139 @@
+# https://github.com/golangci/golangci/wiki/Configuration
```

Comment:
I took this from [one of our other repos](https://bitbucket.org/asecurityteam/aws-accounts-sync/src/master/.golangci.yaml), let me know if there's a specific "golden" one people would prefer